code/__HELPERS/unsorted.dm

```diff
@@ -1175,6 +1175,19 @@ B --><-- A
 
 	return L
 
+/proc/get_areas_in_range(dist=0, atom/center=usr)
+	if(!dist)
+		var/turf/T = get_turf(center)
+		return T ? list(T.loc) : list()
+	if(!center)
+		return list()
+
+	var/list/turfs = RANGE_TURFS(dist, center)
+	var/list/areas = list()
+	for(var/V in turfs)
+		var/turf/T = V
+		areas |= T.loc
+	return areas
```

Comment:
the proc looks good but please dont just shove it into unsorted.dm
put it in game.dm with get_area and cush, or even make a new area.dm and use that